runelite-client/src/main/java/net/runelite/client/plugins/roguesden/RoguesDenPlugin.java

```diff
@@ -25,30 +25,24 @@
 package net.runelite.client.plugins.roguesden;
 
 import com.google.common.eventbus.Subscribe;
-import java.time.temporal.ChronoUnit;
-import java.util.HashMap;
-import javax.inject.Inject;
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.InventoryID;
-import net.runelite.api.Item;
-import net.runelite.api.ItemContainer;
-import static net.runelite.api.ItemID.MYSTIC_JEWEL;
-import net.runelite.api.Tile;
-import net.runelite.api.TileObject;
-import net.runelite.api.events.GameObjectChanged;
-import net.runelite.api.events.GameObjectDespawned;
-import net.runelite.api.events.GameObjectSpawned;
-import net.runelite.api.events.GameStateChanged;
-import net.runelite.api.events.GroundObjectChanged;
-import net.runelite.api.events.GroundObjectDespawned;
-import net.runelite.api.events.GroundObjectSpawned;
+import net.runelite.api.*;
```

Comment:
Don't use wildcard imports, edit settings as described [here](https://github.com/runelite/runelite/wiki/Code-Conventions) to prevent IntelliJ from automatically replacing.